python/lancedb/table.py

```diff
@@ -438,6 +438,8 @@ def search(
             the table
         vector_column_name: str
             The name of the vector column to search.
+
+            The vector column needs to be a "pyarrow.FixedSizeList" type
```

Comment:
Hmm, this wording makes it seem like `pyarrow.FixedSizeList` is some kind of python class but it isn't.  How about just pyarrow fixed size list type (without the double quotes)?